seatunnel-connectors-v2/connector-file/connector-file-hadoop/pom.xml

```diff
@@ -40,4 +40,44 @@
             <artifactId>flink-shaded-hadoop-2</artifactId>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <!-- base module need skip shading -->
+                        <configuration>
+                            <relocations>
+                                <relocation>
+                                    <pattern>org.apache.avro</pattern>
+                                    <shadedPattern>${seatunnel.shade.package}.org.apache.avro</shadedPattern>
```

Comment:
Will `${seatunnel.shade.package}.org.apache.avro` conflict with other connector shade class name?

Reply:
Use the prefix `${seatunnel.shade.package}.connector.file.hadoop.org.apache.avro`?

Reply:
> Will `${seatunnel.shade.package}.org.apache.avro` conflict with other connector shade class name?

Yes, it will conflict with spark.

Reply:
Use the prefix ${seatunnel.shade.package}.${connector}.org.apache.avro?

reference
https://github.com/apache/flink/blob/master/flink-connectors/flink-sql-connector-hive-2.3.9/pom.xml#L103

Reply:
I'll change it when I'm free in the evening.